src/vs/editor/contrib/inlineCompletions/browser/inlineCompletionsController.ts

```diff
@@ -56,6 +57,13 @@ export class InlineCompletionsController extends Disposable {
 		}
 	));
 	private readonly _enabled = observableFromEvent(this.editor.onDidChangeConfiguration, () => this.editor.getOption(EditorOption.inlineSuggest).enabled);
```

Comment:
I would just change this:

```suggestion
	private readonly _enabledInConfig = observableFromEvent(this.editor.onDidChangeConfiguration, () => this.editor.getOption(EditorOption.inlineSuggest).enabled);
        private readonly _enabled = derived(this, reader => this._enabledInConfig.read(reader) && (!this._isScreenReaderEnabled.read(reader) || !this._editorDictationInProgress.read(reader)));
```

(and keep the two newly added observables)

Then everything should work as expected (no other code change needed).